src/app.ts

```diff
@@ -165,8 +160,8 @@ export class CeramicAnchorApp {
    * @private
    */
   private async _startScheduler(): Promise<void> {
```

Comment:
Feels like we might want at least the healthcheck controller for the scheduler service?  At least let's make sure to confirm with platform that we aren't already depending on it.

Reply:
Do not quite understand this comment. The PR does not change the relationship between the services (apart removing some clearly unused dependency in some class).

Reply:
We used to always register the healthcheck controller, including for the scheduler, but now we don't. Unless I'm missing where the HealthcheckController gets registered when running the scheduler?

Reply:
Okay, now I understand. I think, that nothing has changed with regards to how controllers start. Registration in a DI container is no more than a promise of future initialization. For greater confidence, going to create a separate branch to see if the controller in question is ever started in the scheduler mode.

Reply:
okay sounds good, I also saw your discord message confirming that platform isn't using it currently, so seems like it should be fine either way